posthog/queries/session_recordings/test/test_session_recording_list.py

```diff
@@ -105,11 +105,12 @@ def test_all_sessions_recording_object_keys(self):
             self.assertEqual(session_recordings[0]["end_time"], self.base_time + relativedelta(seconds=30))
             self.assertEqual(session_recordings[0]["duration"], 30)
 
+        @test_with_materialized_columns(["$session_id", "$window_id"])
```

Comment:
I don't think we should need these decorators. It runs the test both with and without the columns being materialized, but in this case it's always materialized.